### ISSUE TITLE
Keep getting this error when syncing from Sketch to Avocode.  Using stable version of plugin, not beta.

### ISSUE THREAD

**Issue by brandtreed** [avocode/avocode#655]
IMPORTANT: Please try to describe how did it happen.

-----
Sketch Exporter 2.1.5 running in Sketch 3.5.1
```
Error: Error: Command failed: 2016-01-28 14:00:07.396 sketchtool[35034:8149792] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles)

**Comment by brandtreed** [comment]
I'm running OSX 10.11.2.  I tried syncing UIs from sketch to Avocode via the Beta plugin.  I give me a successful upload message, but when I go to the project in desktop app, there are no new uploads.  Help please! Thanks

**Comment by rychlis** [comment]
We have released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

We're working on updating the older Sketch Plugin 2 however there has been some further issues with Sketch 3.5 unfortunately. You should be able to use the beta version fine though. Please report any issues with prerelease4 as a new issue

Follow #609 for more updates about older Sketch plugin, I am closing this issue as a duplicate, thanks for the report and sorry for the inconvenience
